tests/components/emulated_hue/test_hue_api.py

```diff
@@ -283,12 +282,6 @@ async def test_light_without_brightness_supported(hass_hue, hue_client):
     assert light_without_brightness_json["state"][HUE_API_STATE_ON] is True
     assert light_without_brightness_json["type"] == "On/Off light"
 
-    # BRI required for alexa compat
-    assert (
-        light_without_brightness_json["state"][HUE_API_STATE_BRI]
-        == HUE_API_STATE_BRI_MAX
-    )
-
 
```

Comment:
Untested, and will need to be adjusted.  Something like this should work:

```suggestion
async def test_lights_all_dimmable(hass, hue_client):
    """Test CONF_LIGHTS_ALL_DIMMABLE."""
    # create a lamp without brightness support
    hass.states.async_set("light.no_brightness", "on", {})

    await setup.async_setup_component(
        hass, http.DOMAIN, {http.DOMAIN: {http.CONF_SERVER_PORT: HTTP_SERVER_PORT}}
    )
    await hass.async_block_till_done()

    with patch("homeassistant.components.emulated_hue.create_upnp_datagram_endpoint"):
        await setup.async_setup_component(
            hass,
            emulated_hue.DOMAIN,
            {
                emulated_hue.DOMAIN: {
                    emulated_hue.CONF_LISTEN_PORT: BRIDGE_SERVER_PORT,
                    emulated_hue.CONF_EXPOSE_BY_DEFAULT: True,
                    emulated_hue.CONF_LIGHTS_ALL_DIMMABLE: True,
                }
            },
        )
        await hass.async_block_till_done()

    light_without_brightness_json = await perform_get_light_state(
        hue_client, "light.no_brightness", HTTP_OK
    )

    import pprint

    pprint.pprint(light_without_brightness_json)
    assert light_without_brightness_json["state"][HUE_API_STATE_ON] is True
    assert light_without_brightness_json["type"] == "On/Off light"
    assert (
        light_without_brightness_json["state"][HUE_API_STATE_BRI]
        == HUE_API_STATE_BRI_MAX
    )
```

Reply:
merged with the main commit, applied black